File: linguagens_de_programacao/C/impares.c
#include <stdio.h>

int main()
{
    int num, cont, aux;

    printf("digite o número de ímpares: ");
    scanf("%d", &num);
    printf("os primeiros ímpares são: ");
    
    aux = 0;
    
    for (cont = 0; aux != num; cont++ )
        if (cont%2 != 0) {
            printf("%d ", cont);
            aux++;
        }
    return 0;
}

File: linguagens_de_programacao/go/impares.go
package main

import (
	"fmt"
)

func main() {
	var n int
	fmt.Print("digite um número inteiro positivo: ")
	fmt.Scan(&n)

	fmt.Println("os", n, "primeiros naturais ímpares são:")
	count := 0
	num := 1
	for count < n {
		fmt.Print(num, " ")
		num += 2
		count++
	}
	fmt.Println()
}
